backend/execute_queries.py

```diff
@@ -0,0 +1,5 @@
+import queries.home_video_collection as home
+
+
+
+home.home_video_collection.insert_many(home.video_data)
```

Comment:
you can put this in home_video_collection file itself. Then one can execute home_video_collection.py directly.